src/admin/dumb_components/letter_body.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default ({ letter }) => {
+
+  const formatContent = (content) => (
+    <div>
+      {content.split('\n').map((paragraph, i) =>
```

Comment:
Rafe could you recommend another solution for this? What else could we set the key to?